manifests/database/postgresql.pp

```diff
@@ -59,8 +59,11 @@
     # Only install pg_trgm extension, if database it is actually managed by the module
     if $manage_database {
 
-      # get the pg contrib to use pg_trgm extension
-      class { '::postgresql::server::contrib': }
+      # from postgresql version 10 onwards, this extension is no longer inside the contrib package, but is being bundled with the postgresql package itself
+      if (versioncmp($postgres_version, '10') == -1) {
```

Comment:
```suggestion
      if (versioncmp($postgres_version, '10') < 0) {
```

Better always compare to 0: this allows to read the condition as `$postgres_version < 10` by moving the operator between the parameters of `versioncmp` in your mind :wink: